javascript/ql/src/experimental/Security/CWE-090/LdapInjectionCustomizations.qll

```diff
@@ -31,10 +45,13 @@ module LdapInjection {
   }
 
   /**
-   * An LDAP filter of an API call that executes an operation against the LDAP server.
+   * An LDAP filter for an API call that executes an operation against the LDAP server.
    */
   class LdapjsSink extends Sink {
-    LdapjsSink() { this instanceof LdapjsSearchFilter }
+    LdapjsSink() {
+      this instanceof LdapjsSearchFilter or
+      this instanceof LdapjsDNArgument
+    }
   }
```

Comment:
The two sinks are different, so I would suggest you move them into separate classes (you can see an example of that [in another query](https://github.com/github/codeql/blob/main/javascript/ql/src/semmle/javascript/security/dataflow/CleartextStorageCustomizations.qll#L49-L75)). 

That would also make it easier to implement to implement the `getQueryCall()` on each of the new subclasses. 

When you do it I would suggest you do it like this: 
```CodeQL
class LdapjsSearchFilterAsSink extends Sink, LdapjsSearchFilter {...
```

That way you can easily use the methods defined in `LdapjsSearchFilter` when writing `getQueryCall()`.

Reply:
> It sounds like you added tests, but I don't see any, did you forget to commit them?
> (Again, you don't have to add tests for this PR, it was just a suggestion for how to improve in the future).

sorry for the misunderstanding. What I mean was that I used that link to write test for another CodeQL query (in another PR), but not for this one.